index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HOVER-Library</title>
+    <link href="style.css" rel="stylesheet">
+    </link>
+</head>
+
+
+<body>
+
+    <header>
+        <nav class="logo">
+            <ul class="logo-ul">
+                <li><a class="header-color" href="/index.html">HOVER</a></li>
+                <li><span>v1.0</span></li>
+            </ul>
+        </nav>
+        <nav class="menu">
+
+            <ul class="menu-ul ul-main">
+                <li><a href="index.html">Home</a></li>
+                <li><a href="/components/badge.html">Docs</a></li>
+                <li><a href="https://hover-library.netlify.app/style.css" download>Download</a></li>
+
+                <li><a href="https://github.com/jinath123-yasmin/Hover-Components-CSS-Library">Github</a></li>
```

Comment:
The viewer/user should never leave your page, so what you can do is just add `target = "_blank"` inside `<a>` tag so that a new tab opens when the link is clicked.

Reply:
Okay got it. I will add & Thank you Sneha.